starcli/search.py

```diff
@@ -55,7 +55,8 @@ def search(
     last_updated=None,
     stars=">=100",
     topics=[],
-    debug=False,
+    user=None,
+    debug=True,
```

Comment:
Could you set this to False by default please

Reply:
Apologies, will correct now.